### DISCUSSION TITLE
The model of ocr

### DISCUSSION THREAD

**Discussion by gcusms** [wildwolf-team/WolfVision#31] [General]
@iChizer0  How to add the path of the ocr model which suffix is ".onnx".I can't add the path in the wolfvision.cpp by relative path but absolute path.
For example I use " onnx_inferring::model model_ = onnx_inferring::model(fmt::format("{}{}", CONFIG_FILE_PATH, "/ml/mnist-8.onnxl"));" which can get the target but it can't run as much as the path in the config configs/ml/onnx_inferring_config.xml I add.

The error reports is "terminate called after throwing an instance of 'cv::Exception'
  what():  OpenCV(4.1.0) /home/sms/opencv-4.1.0/modules/dnn/src/onnx/onnx_importer.cpp:57: error: (-210:Unsupported format or combination of formats) Failed to parse onnx model in function 'ONNXImporter'"

**Comment by gcusms** [comment]
@iChizer0如何添加后缀为“.onnx”的ocr模型的路径。我不能按相对路径添加wolfvision.cpp中的路径，而只能按绝对路径添加。
例如，我使用“onnx_inferring:：model model=onnx_inferring:：model（fmt:：format（“{}{}”，CONFIG_FILE_PATH，“/ml/mnist-8.onnxl”）；”它可以获取编译，但不能像我添加的configs/ml/onnx_inferring_config.xml中的相对路径那样运行。

**Reply by fqjun** [reply]
在 CMakeLists.txt 中参考这一句 `set(CONFIG_FILE_PATH ${PROJECT_SOURCE_DIR}/configs)
` 来添加路径地址试试
